src/renderer/components/item-object-list/item-list-layout.tsx

```diff
@@ -45,7 +45,6 @@ export interface ItemListLayoutProps<T extends ItemObject = ItemObject> {
   isClusterScoped?: boolean;
   hideFilters?: boolean;
   searchFilters?: SearchFilter<T>[];
-  filterItems?: ItemsFilter<T>[];
```

Comment:
Are you sure that this is not a breaking change. Shouldn't we mark it as depcricated (or is `ItemListLayout` not exported in 4.0.X?)

Reply:
Yes, it is.. Initially I thought it's not, but:

`export interface KubeObjectListLayoutProps extends ItemListLayoutProps`

and we re-export `KubeObjectListLayout` to extensions-api..

Reply:
So furthermore we cannot remove this functionality at all because that is a breaking change.